src/components/common/Navbar/navbar.scss

```diff
@@ -2,8 +2,8 @@
 
 .navbar {
   padding: 0;
-  border-bottom: var(--border-width-0) solid var(--color-neutral-2);
-  box-shadow: 0 var(--space-0) var(--space-3) var(--color-black-transparent-5);
+  border-bottom: var(--fs-border-width-default) solid var(--fs-color-neutral-2);
+  box-shadow: 0 var(--fs-spacing-0) var(--fs-spacing-3) rgb(0 0 0 / 5%);
```

Comment:
Is this the new approach for colors with transparency? (We won't have an specific token for it?)

Reply:
Oh, no, @hellofanny, we'll absolutely need tokens to deal with transparency. Tokens should be assigned to all component parameters. It's just that I wasn't sure where to put it (or what to call it) at the time.